zowe-install/src/main/resources/component-scripts/start.sh

```diff
@@ -112,14 +112,14 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${GATEWAY_CODE} java -Xms32m -Xmx256m -Xquickstart \
     -Dapiml.service.hostname=${ZOWE_EXPLORER_HOST} \
     -Dapiml.service.port=${GATEWAY_PORT} \
     -Dapiml.service.discoveryServiceUrls=https://${ZOWE_EXPLORER_HOST}:${DISCOVERY_PORT}/eureka/ \
-    -Dapiml.service.preferIpAddress=true \
-    -Dapiml.service.allowEncodedSlashes=true \
-    -Dapiml.service.corsEnabled=false \
+    -Dapiml.service.preferIpAddress=${APIML_PREFER_IP_ADDRESS} \
+    -Dapiml.service.allowEncodedSlashes=${APIML_ALLOW_ENCODED_SLASHES} \
+    -Dapiml.service.corsEnabled=${APIML_CORS_ENABLED} \
     -Dapiml.cache.storage.location=${WORKSPACE_DIR}/api-mediation/ \
     -Denvironment.ipAddress=${ZOWE_IP_ADDRESS} \
-    -Dapiml.gateway.timeoutMillis=600000 \
+    -Dapiml.gateway.timeoutMillis=${APIML_GATEWAY_TIMEOUT_MILLIS} \
     -Dapiml.security.ssl.verifySslCertificatesOfServices=${VERIFY_CERTIFICATES} \
-    -Dapiml.security.auth.zosmfServiceId=zosmf \
+    -Dapiml.security.auth.zosmfServiceId=${APIML_SECURITY_AUTH_ZOSMF_SERVICE_ID} \
```

Comment:
Can the user that install Zowe influence the zosmf service id?

Reply:
Good point. I've to double check